### PR TITLE
svelte-img implementation

### LINKED ISSUE #3: Use svelte-img for perf
https://github.com/zerodevx/svelte-img

### PR DESCRIPTION
**Pull request by KiritoDv**
Closes #3

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```
Comment (bates64) [inline pnpm-lock.yaml:1]: Update your pnpm and run `pnpm i` again pls

File: src/routes/LandingHeader.svelte [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+    import logotype from "$lib/assets/logotype.png?run&lqip=0";
+    import Img from '@zerodevx/svelte-img'
 </script>
 
 <header class="align-center flex items-center p-4">
     <div class="max-w-2xl">
         <div class="flex items-center justify-center gap-2 text-white">
             <h1 id="name" class="lg text-6xl font-bold sm:text-7xl lg:text-8xl">
-                <img src={logotype} alt="Star Haven" />
+                <Img src={logotype} alt="Star Haven"/>
```
Comment (bates64) [inline src/routes/LandingHeader.svelte]: does this pass prettier?

File: pnpm-lock.yaml (hunk 1 above)
Comment (KiritoDv) [reply pnpm-lock.yaml:1]: It doesnt update :(

**Review by bates64 — APPROVED** [review]

**Review by bates64 — CHANGES_REQUESTED** [review]
[/games has images 404ing](https://starhaven-dot-dbtfnybzn-imalex.vercel.app/games) - thanks to @L5050 for finding this

File: src/routes/games/games.ts [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+import csbox from "./cs/3dbox.webp?run";
+import okbox from "./ok/3dbox.webp?run";
 
 export interface Game {
     url?: string;
     viewTransitionTag: string;
     name: string;
     console: string;
-    boxArtUrl: string;
+    boxArtUrl: object[];
```
Comment (bates64) [inline src/routes/games/games.ts]: Is there a better type than `object` here? :/

Comment (KiritoDv) [reply src/routes/games/games.ts]: I changed to object because on the svelte-img doc, the ambient defines it as an Object[]

Comment (KiritoDv) [reply src/routes/games/games.ts]: A custom type could work?

File: src/ambient.d.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/ban-types */
+// Squelch warnings of image imports from your assets dir
+
+type AssetData = Object[];
```
Comment (bates64) [inline src/ambient.d.ts]: Can you make this an [opaque newtype](https://blog.beraliv.dev/2021-05-07-opaque-type-in-typescript)?
```suggestion
declare const _opaque = unique symbol;
type Opaque<T, V> = T & { readonly [_opaque]: V }

type AssetData = Opaque<Object[], "AssetData">;
```

Comment (KiritoDv) [reply src/ambient.d.ts]: Sure

**Review by bates64 — APPROVED** [review]
LGTM including /games

**Pull request merged**